csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -300,10 +296,7 @@ public bool Remove(T item)
         /// <param name="values">The values to add to this collection.</param>
         public void Add(RepeatedField<T> values)
         {
```

Comment:
Yes, could do that. Or we could remove the overload for Add, and just detect it as another kind of  optimization in AddRange. Very happy to do that.